frame/support/src/traits/tokens/fungibles.rs

```diff
@@ -281,9 +275,41 @@ pub trait Destroy<AccountId>: Inspect<AccountId> {
 	///
 	/// If successful, this function will return the actual witness data from the destroyed asset.
```

Comment:
this part of the doc also needs to be updated

Reply:
Yeah. Thanks again. Also taken it out now.